relay/pushservice/pushservice.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import messaging
+
+from relay.events import Event
+from relay.api.schemas import UserCurrencyNetworkEventSchema
+from .client_token_db import ClientTokenDB
+
+
+class FirebaseRawPushService:
+    """Sends push notifications to firebase. Sending is done based on raw client tokens"""
+
+    def __init__(self) -> None:
+        self._app = None
```

Comment:
should probably be self.app = None. it least this is what is being used later.

Reply:
Yep that with the app is a mistake.
I can move it into init, but I usually try to have only simple code in init and this seemed to me to much stuff for the init method as I dont know if this need network traffic.

Reply:
I see your point. But doing it this way, is putting the burden on the caller, who must now call an additional method in order to get a fully initialized object. And he does, right after creating the object, so the caller doesn't have a need to have 'delayed object initialization'. He could provide all the needed arguments for a full initialization.

If you really need that kind of delayed initialization, I think the class itself should encapsulate it, i.e. it could store the needed parameters for initialization of self._app and create that on the 'first real use'.

But at the moment I think it would be enough to just initialize everything in `__init__`.  It's similar to what is being done now anyway.

Reply:
Okay, will do that